docs/develop/contributing.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+Contributions are welcome and much appreciated. Every little bit helps. You can contribute by improving the documentation, adding missing features, and fixing bugs. You can also help out by reviewing and commenting on [existing issues][].
+
+## Development Environment
+
+AutoPub is built with [Poetry][], so if you don’t already have it installed, the first step is to install it by following the [Poetry Installation Documentation][].
+
+Once [Poetry][] is installed, follow the steps below to install AutoPub for development:
```

Comment:
links are missing here :)

Reply:
These are called "reference-style links", and this particular type uses the "implicit link name shortcut". If you look at the bottom of the file, you'll see the links that correspond to these references. More details: [Official Markdown Specification > Links](https://daringfireball.net/projects/markdown/syntax#link).

Reply:
My pleasure! Sometimes I find that long URLs in inline links break the flow, so putting the URLs at the bottom can often improve readability substantially. Also, as in this case, this method allows you to specify the URL once but link multiple references to it, which is much more DRY than peppering the text with duplicate URLs.